src/6_branch.js

```diff
@@ -210,11 +210,12 @@ Branch.prototype._referringLink = function() {
 /***
  * @function Branch._publishEvent
  * @param {string} event
+ * @param {Object} data - _optional_ - data to pass into listener callback.
  */
-Branch.prototype._publishEvent = function(event) {
+Branch.prototype._publishEvent = function(event, data) {
 	for (var i = 0; i < this._listeners.length; i++) {
 		if (!this._listeners[i].event || this._listeners[i].event === event) {
-			this._listeners[i].listener(event);
+			this._listeners[i].listener(event, data);
 		}
 	}
 };
```

Comment:
Might want to add banner_id to https://github.com/BranchMetrics/web-branch-deep-linking/blob/master/src/6_branch.js#L1557 (the event that is published for didCallJourneyClose) as well